docs/make.jl

```diff
@@ -1,6 +1,6 @@
-using Documenter, Nemo, AbstractAlgebra
+using Documenter, Nemo, Nemo.AbstractAlgebra
 
-DocMeta.setdocmeta!(Nemo, :DocTestSetup, :(using Nemo); recursive = true)
+DocMeta.setdocmeta!(Nemo, :DocTestSetup, :(using Nemo; AbstractAlgebra = Nemo.AbstractAlgebra); recursive = true)
```

Comment:
Not an objection, just a genuine question: I am wondering where we need this? Is it because of the doctests in AA which have `setup = :(using AbstractAlgebra)`? If so, is that the only reason? Perhaps we could add this as a comment before the `setdocmeta!`, for the next time I or someone else wonders about this ;-). 

But if you don't know either, don't waste your time on this.

Reply:
Yeah, that was my hope, but it doesn't work in the docs ci job.

Reply:
I'll try something else later today